source/globals.d.ts

```diff
@@ -57,6 +57,7 @@ declare namespace JSX {
 		'details-menu': BaseIntrinsicElement & {src?: string; preload?: boolean};
 		'time-ago': BaseIntrinsicElement & {datetime: string; format?: string};
 		'relative-time': BaseIntrinsicElement & {datetime: string};
+		'clipboard-copy': BaseIntrinsicElement & {role?: string};
```

Comment:
`role` is a native attribute though, perhaps it should extend `HTMLButtonElement` instead of `div`

Reply:
hmmm
```
'clipboard-copy' components don't accept text as child elements. Text in JSX has the type 'string', but the expected type of 'children' is 'HTMLCollection'.
```

Reply:
You can just set `HTMLButtonElement` there, it's probably `IntrinsicElements['button']`